src/js/components/Meter.js

```diff
@@ -117,7 +118,8 @@ var Meter = React.createClass({
       type: 'bar',
       a11yRole: 'img',
       a11yTitleId: 'meter-title',
-      a11yDescId: 'meter-desc'
+      a11yDescId: 'meter-desc',
+      indicators:false,
```

Comment:
there is an extra comma here and the spacing is off. I believe it should be `indicators: false`